src/js/components/banner.js

```diff
@@ -0,0 +1,16 @@
+var select = require('../utils/select');
+var addClass = require('../utils/add-class');
+var removeClass = require('../utils/remove-class');
+var dispatch = require('../utils/dispatch');
+
+var headers = select('.usa-banner-header');
+headers.forEach(function (header) {
+  var click = function (e) {
```

Comment:
I _think_ we're avoiding use of `Event.preventDefault()` because it's not fully supported in older IEs, but having said that, I'm not entirely sure of the story. I've tended to switch them to `return false;` (at the end of the function) even though that's not quite the same thing.

I'll look into it and see if I can get a more solid tactic here.